.github/workflows/update_preview.yml

```
@@ -0,0 +1,34 @@
+name: "Build Preview"
+on:
+  push:
+    branches:
+      - main
+      - project-*
+  pull_request:
+
+jobs:
+  update_pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: ./.github/actions/setup-environment
+
+      - name: Build
+        run: |
+          npm run storybook:build -- --preview-url ./
+          npm run browser:build -- --base ./
+
+          mkdir dist_preview
+          mkdir dist_preview/editor
+          mkdir dist_preview/storybook
+          cp -r dist/* dist_preview/editor/
+          cp -r storybook-static/* dist_preview/storybook/
```

結構なファイル数になる＆`cp *`だと隠しファイルがされないので、ディレクトリごとmvのがいいかも。

こうとか！
```suggestion
      - name: Build
        run: |
          mkdir dist_preview

          npm run storybook:build -- --preview-url ./
          mv storybook-static/ dist_preview/storybook/

          npm run browser:build -- --base ./
          mv dist/ dist_preview/editor/
```